### ISSUE TITLE
Full screen broken when multiple screens on Linux (Arch)(scaled by the total size)

### ISSUE THREAD

**Issue by potens1** [mickelson/attract#616]
Hello,

First of all, this software seems huge ! Thank you for that !

It seems the fullscreen mode is broken on multiple screen on Linux. when I start attract in full screen, it does full screen but the content of the screen seems to cover only 1/(number of screens) of the area, the remaining space is black, so, i.e on my 3 screens setup, I get something like (behold of the ASCII art) this:
```
+--------------------+
|Attr  |  2*empty    |
+--------------------+
```
with Attr being the part effectively displaying Attract, squeezed to 1/3 in width (height is good)

I've tried to option called "multi screen" but it does not seems to change anything, and I don't know if it is designed for that. 

I'm using Arch linux and installed attract with the aur file from [here](https://aur.archlinux.org/packages/attract/), attract 2.6.1, libxinerama 1.1.4, and nvidia-drivers 440.82.

I did not find what level of details you expect for bug report, so just ask me if something is missing

**Comment by mickelson** [comment]
yes I suspect multimonitor support is hopelessly broken in many cases... unfortunately I don't have a set up to test multimon at the moment.
